src/rebuild/srv.c

```diff
@@ -2044,28 +2155,25 @@ rebuild_ults(void *arg)
 	ABT_cond_signal(rebuild_gst.rg_stop_cond);
 	rebuild_gst.rg_rebuild_running = 0;
 	ABT_mutex_unlock(rebuild_gst.rg_lock);
+	D_INFO("exiting\n");
 }
 
 void
 ds_rebuild_abort(uuid_t pool_uuid, unsigned int ver, unsigned int gen, uint64_t term)
 {
 	struct rebuild_tgt_pool_tracker *rpt;
 
-	rebuild_leader_stop(pool_uuid, ver, gen, term);
+	rebuild_leader_abort(pool_uuid, ver, gen, term);
 
 	/* Only stream 0 will access the list */
 	while(1) {
 		bool aborted = true;
 
-		d_list_for_each_entry(rpt, &rebuild_gst.rg_tgt_tracker_list, rt_list) {
-			if (uuid_compare(rpt->rt_pool_uuid, pool_uuid) == 0 &&
-			    (ver == (unsigned int)(-1) || rpt->rt_rebuild_ver == ver) &&
-			    (gen == (unsigned int)(-1) || rpt->rt_rebuild_gen == gen) &&
-			    (term == (uint64_t)(-1) || rpt->rt_leader_term == term)) {
-				D_INFO(DF_RB " try abort rpt %p\n", DP_RB_RPT(rpt), rpt);
-				rpt->rt_abort = 1;
-				aborted = false;
-			}
+		rpt = rpt_lookup(pool_uuid, -1 /* opc */, ver, gen);
```

Comment:
why change the original code here?
as the original func take the "term" as the parameter, looks better to keep the original code.
although it should be same if called as "ds_rebuild_abort(pool->sp_uuid, -1, -1, -1)", but if call with not -1 term, the original code is better for that case.